src/main/resources/com/nirima/jenkins/plugins/docker/DockerTemplate/help-connector.html

```diff
@@ -0,0 +1,31 @@
+<div>
+
+    <p>Basically, for all connection methods, Jenkins will start by triggering a <code>docker run</code>.
```

Comment:
While I, too, have a habit of starting many an explanation with the word "Basically", I think we can remove it in text.  It's horribly informal.
Maybe "In summary" instead?